far2l/src/vt/vtshell.cpp

```diff
@@ -829,10 +832,11 @@ class VTShell : VTOutputReader::IProcessor, VTInputReader::IProcessor, IVTShell
 			}
 
 			if (!KeyEvent.bKeyDown)
-				return std::string();
```

Comment:
hm, why? theoretically return std::string() is more efficient than return std::string("")

Reply:
fixed